docs/samples/Microsoft.ML.Samples/Dynamic/DataOperations/EntityFrameworkEnumerable.cs

```diff
@@ -0,0 +1,128 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using Microsoft.EntityFrameworkCore;
+
+namespace Microsoft.ML.Samples.Dynamic
+{
+    public class HousingData
+    {
+        // Entity Framework requires the object to have a primary key
+        public int HousingDataId { get; set; }
+        public float MedianHomeValue { get; set; }
+        public float CrimesPerCapita { get; set; }
+        public float PercentResidental { get; set; }
+        public float PercentNonRetail { get; set; }
+        public float CharlesRiver { get; set; }
+        public float NitricOxides { get; set; }
+        public float RoomsPerDwelling { get; set; }
+        public float PercentPre40s { get; set; }
+        public float EmploymentDistance { get; set; }
+        public float HighwayDistance { get; set; }
+        public float TaxRate { get; set; }
+        public float TeacherRatio { get; set; }
+    }
+
+    public partial class MLNetExampleContext : DbContext
+    {
+        public DbSet<HousingData> Housing { get; set; }
+
+        public MLNetExampleContext(DbContextOptions<MLNetExampleContext> options)
+            : base(options)
+        {
+        }
+
+        protected override void OnConfiguring(DbContextOptionsBuilder optionsBuilder)
+        {
+            if (!optionsBuilder.IsConfigured)
+            {
+                optionsBuilder.UseSqlServer("Server=(localdb)\\mssqllocaldb;Database=EFProviders.InMemory;Trusted_Connection=True;ConnectRetryCount=0");
```

Comment:
Worth adding some external configuration files to store the connection string or at least a TO-DO comment here acknowledging that in a real world application this shouldn't be hardcoded and checked in as part of the code.